File: 0789 - Fundamentos de linguagem JAVA/semana5/demoClasses/Pessoa.py
from dataclasses import dataclass

@dataclass
class Pessoa:
    # Atributos
    nome: str
    idade: int
    genero: str

    # métodos

    def envelhecer(self, anos: int = 1):
        self.idade += anos # self.idade = self.idade + anos
        # i++ <- i += 1

File: 5423 - Acesso móvel a sistemas de informação/aula_5/aula_5/Pessoa.swift
//
//  Pessoa.swift
//  aula_5
//
//  Created by Gonçalo Feliciano on 22/10/2024.
//

class Pessoa{
    
    var nome: String
    var idade: Int
    
    init(nome: String, idade: Int) {
        self.nome = nome
        self.idade = idade
    }
    

    
    func envelhercer(anos: Int = 1){
        self.idade += anos
    }
    
    
    
    final func infos2() -> String{
        return "Pessoa"
    }
    
    func infos() -> String{
        return "Pessoa"
    }
    
}
